openshift-ci/Dockerfile.registry.build

```diff
@@ -1,24 +1,14 @@
 FROM quay.io/openshift/origin-operator-registry:latest
 
+ARG image=registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:devconsole-operator
 ARG version=0.1.0
 
 COPY manifests manifests
 COPY deploy/crds/*.yaml manifests/devconsole/${version}/
-RUN curl -L -s https://github.com/devconsole-operator-test-deps/operator-registry-binaries/releases/download/v0.1.0/appregistry-server -o /bin/appregistry-server \
```

Comment:
Let's keep only these changes and have this PR merged?

Reply:
Done. 697d40640b29644386afa4d44424d661f819603e